src/chrome/content/rules/Criosweb.xml

```diff
@@ -0,0 +1,47 @@
+<ruleset name="Criosweb (partial)">
+
+	<!--	MCB: https://infoeuri.criosweb.ro/blog/	-->
+
+	<target host="infoeuri.criosweb.ro" />
+
+	<rule from="^http://infoeuri\.criosweb\.ro/$"
+		to="https://infoeuri.criosweb.ro/" />
+		<test url="http://infoeuri.criosweb.ro/" />
+
+	<rule from="^http://infoeuri\.criosweb\.ro/(aplicatie|css|fonts|img|js|povestea\.php)/"
+		to="https://infoeuri.criosweb.ro/$1/" />
+		<test url="http://infoeuri.criosweb.ro/aplicatie/index.html/" />
+		<test url="http://infoeuri.criosweb.ro/css/jumbotron.css/" />
+		<test url="http://infoeuri.criosweb.ro/fonts/glyphicons-halflings-regular.woff/" />
+		<test url="http://infoeuri.criosweb.ro/img/favicon.png/" />
+		<test url="http://infoeuri.criosweb.ro/js/jquery-1.js/" />
+		<test url="http://infoeuri.criosweb.ro/povestea.php/" />
+
+	<rule from="^http://infoeuri\.criosweb\.ro/blog/wp-(content|includes)/"
+		to="https://infoeuri.criosweb.ro/blog/wp-$1/" />
+		<test url="http://infoeuri.criosweb.ro/blog/wp-content/uploads/2014/07/august-1038x576.jpg" />
+		<test url="http://infoeuri.criosweb.ro/blog/wp-includes/js/tw-sack.min.js" />
+
+	<!--	MCB: https://luci.criosweb.ro/	-->
+
+	<target host="luci.criosweb.ro" />
+
+	<exclusion pattern="^http://luci\.criosweb\.ro/$" />
+	<rule from="^http://luci\.criosweb\.ro/(blog/)?wp-(content|includes)/"
+		to="https://luci.criosweb.ro/$1wp-$2/" />
+		<test url="http://luci.criosweb.ro/" />
+		<test url="http://luci.criosweb.ro/blog/wp-content/themes/twentyfifteen/genericons/genericons.css" />
+		<test url="http://luci.criosweb.ro/blog/wp-includes/js/jquery/jquery-migrate.min.js" />
+		<test url="http://luci.criosweb.ro/wp-content/uploads/2015/11/board-config.png" />
+		<test url="http://luci.criosweb.ro/wp-includes/js/tw-sack.min.js" />
+
+	<!--	http://criosweb.ro/$ auto redirects to http://www.criosweb.ro/$ by serve.	-->
+	
+	<target host="criosweb.ro" />
+	<target host="www.criosweb.ro" />
+
+	<rule from="^http://(www\.)?criosweb\.ro/" to="https://www.criosweb.ro/" />
```

Comment:
@J0WI I have add the comment:
	<!--	http://criosweb.ro/$ auto redirects to http://www.criosweb.ro/$ by serve.	-->